MmSupervisorPkg/Core/Mem/SmmCpuMemoryManagement.c

```diff
@@ -563,14 +581,21 @@ ConvertMemoryPageAttributes (
     return RETURN_SUCCESS;
   }
 
-  PageTableBufferSize = 0;
-  WRITE_UNPROTECT_RO_PAGES (WriteProtect, CetEnabled);
-  Status = PageTableMap (&PageTableBase, PagingMode, NULL, &PageTableBufferSize, BaseAddress, Length, &PagingAttribute, &PagingAttrMask, IsModified);
+  while (!SamePageTable) {

```

Comment:
Will we get stuck here forever if we cannot get more pages for page tables?

Reply:
That's correct but that behavior can be changed if that's preferred.  Do we want to continue booting if it fails?